.github/workflows/promptflow-sdk-cli-test.yml

```diff
@@ -76,14 +114,12 @@ jobs:
       working-directory: ${{ env.testWorkingDirectory }}
       run: |
         gci env:* | sort-object name
-        az account show
         python "../../scripts/building/run_coverage_tests.py" `
-          -p promptflow `
-          -t ${{ github.workspace }}/src/promptflow/tests/sdk_cli_azure_test ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test `
-          -l eastus `
-          -m "unittest or e2etest" `
-          -n ${{ steps.cpu-cores.outputs.count }} `
-          --coverage-config ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test/.coveragerc
+        -p promptflow `
+        -t ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test `
+        -l eastus `
+        -m "unittest or e2etest" `
+        --coverage-config ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test/.coveragerc
     - name: Upload Test Results
```

Comment:
Does this step upload both of 3.8 and 3.9 results?

Reply:
![image](https://github.com/microsoft/promptflow/assets/2208599/2ac6c677-4f7c-42da-8382-8e788824395c)

Reply:
Yes. This is the same technique we use in the testing matrix.

Reply:
Seems python 3.8's output not appear?
![image](https://github.com/microsoft/promptflow/assets/7776147/6f649860-582e-4461-bc94-c1e7de12675a)